File: BKP/TrainingDataAccess/Models/Activity.cs
namespace TrainingDataAccess.Models
{
    public class Activity //: IEquatable<Activity>
    {
        //[Key] public int ActivityId { get; set; }

        public string Name { get; set; }
        public string Description { get; set; }
        public int? PersonsMin { get; set; }
        public int? PersonsMax { get; set; }
    }
}

File: FloorballTraining.Client/src/app/shared/models/Activity.ts
import { AgeGroup } from "./agegroup";
import { Place } from "./place";
import { Tag } from "./tag";

export interface Activity {
  id: number;
  name: string;
  description: string;
  personsMin: number;
  personsMax: number;
  durationMin: number;
  durationMax: number;
  intensity: number;
  difficulty: number;
  placeWidth: number;
  placeLength: number;
  environment: string;

  tags: Tag[];
  places: Place[];
  ageGroups: AgeGroup[];
}

